pwndbg/commands/tls.py

```diff
@@ -24,6 +24,7 @@
 
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.LINUX)
 @pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithLibraryLoaded("libc.so")
```

Comment:
Hmm I wonder: do errno or tls command work in statically linked binaries? (cc: @lebr0nli)

If they do, maybe we do not need the decorator/check?

Reply:
> Hmm I wonder: do errno or tls command work in statically linked binaries? (cc: @lebr0nli)

Yes, `errno` and `tls` might also work in statically linked binaries.

> If they do, maybe we do not need the decorator/check?

Yeah, currently I think it probably needs more consideration before applying this decorator, to avoid accidentally blocking the user from using the commands.

Reply:
reasonable - so would you like me to prepare a pull request that just adds the unused decorator or not really?

Reply:
Not really, lets close this for now